Makefile

```diff
@@ -6,15 +6,25 @@ LDFLAGS := -ldflags="-s -w -X \"main.Version=$(VERSION)\" -X \"main.Revision=$(R
 
 .DEFAULT_GOAL := bin/$(NAME)
 
-bin/$(NAME):
+bin/$(NAME): deps
```

Comment:
I think `make deps` should be executed independently, because `make deps` spends time even if there is no dependency difference.

Instead, watching all Go sources is better. When there is no source difference, `go build` will not be invoked.

```makefile
SRCS := $(shell find . -name '*.go' -type f)

bin/$(NAME): $(SRCS)
```

## REF

[Go でツール書くときの Makefile 晒す - Qiita](http://qiita.com/dtan4/items/8c417b629b6b2033a541)